### PR TITLE
add docs for custom action and swipes

### PR DESCRIPTION
**Pull request by AlexeyKorshun**
update Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,6 +128,67 @@ lifecycleScope.launch {
...
+    elevation = 1f,
+    cornerRadius = 2f,
+    drawableOffsetInDp = 24,
+)
+```
+
+and add it to `itemDismissCallbacks` in you builder function for TiRecycler
+
+```kotlin
+TiRecyclerCoroutines<ViewTyped>(recyclerView, holderFactory) { //for RX use similar function builder
```
Comment (Jacks0N23) [inline README.md:151]: pls, write comment above the code, not inside

File: README.md [hunk 2]
```diff
@@ -128,6 +128,67 @@ lifecycleScope.launch {
...
+    itemDismissCallbacks += ItemDismissTouchHelperCallback()
+}
+```
+
+and after it, you can collect events via `recycler.swipeToDismiss` method
+
+# How to use custom actions
+
+You need to implement `TiRecyclerCustomActionFlow/TiRecyclerCustomActionObservable` after it should add to map of you
+holder factory
```
Comment (Jacks0N23) [inline README.md:162]:
```suggestion
You need:
* implement your own action with extends from `TiRecyclerCustomActionFlow/TiRecyclerCustomActionObservable` 
* add it to map in your holder factory
```

File: README.md (hunk 1 above)
Comment (AlexeyKorshun) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (AlexeyKorshun) [reply README.md]: done

**Comment by AlexeyKorshun** [comment]
also add samples for custom actions in sample app

**Review by Jacks0N23 — APPROVED** [review]

**Pull request merged**